ZionEscape/MessageBox.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#ifndef _MESSAGEBOX_H_
+#define _MESSAGEBOX_H_
+
+#include "BitmapManager.h"
+
+ref class Messagebox {
+  bool activated;
+  BitmapManager^ bmpManager;
+  int counter, len, order, nextMessage;
+  String^ txt;
+public:
+  Messagebox() {
+    this->bmpManager = BitmapManager::GetInstance();
+    this->counter = len = order = nextMessage = 0;
+  }
+  ~Messagebox() {
+    delete this->bmpManager;
+    delete this->txt;
+  }
+  void Draw(Graphics^ g) {
+    //Select the image and the position
+    g->DrawImage(bmpManager->GetImage("assets\\sprites\\misc\\messagebox.png"), Point(101,230));
+  }
```

Comment:
Aquí puedes obtener la instancia del `BitmapManager` y usarlo.

```suggestion
  void Draw(Graphics^ g) {
    BitmapManager^ bmpManager = BitmapManager::GetInstance();
    //Select the image and the position
    g->DrawImage(bmpManager->GetImage("assets\\sprites\\misc\\messagebox.png"), Point(101,230));
  }
```

Incluso, podrías hacerlo todo en una sola línea (pero se ve feo):

```suggestion
  void Draw(Graphics^ g) {
    //Select the image and the position
    g->DrawImage(BitmapManager::GetInstance()->GetImage("assets\\sprites\\misc\\messagebox.png"), Point(101,230));
  }
```